style.css

```diff
@@ -307,44 +338,53 @@ body {
   justify-content: center;
   align-items: stretch;
   text-align: center;
-  list-style: none;
+  padding-bottom: 70px;
 }
 
 .skills-title {
+  display: flex;
+  flex-direction: column;
+  gap: 44px;
+}
+
+.skill-ul {
   width: 100%;
   margin: 10px 0;
   display: flex;
-  flex-direction: column;
   justify-content: center;
   align-items: center;
   gap: 12px;
 }
 
 .skills-type {
-  font-size: 32px;
+  font-size: 25px;
   color: #fff;
+  font-family: "Inter", sans-serif;
+  font-weight: 700;
 }
 
 .skills-list {
   color: #fff;
   margin-top: 2rem;
   align-self: center;
   text-align: center;
-  list-style: none;
   display: flex;
   flex-direction: column;
   gap: 32px;
+  list-style: none;
 }
 
 .list-item {
-  font-weight: 400;
   font-size: 22px;
+  text-align: center;
+  font-weight: 400;
 }
 
 .noTagList {
   list-style: none;
 }
 
+/* contact section */
```

Comment:
[OPTIONAL] kindly do your best to replicate figma design the title in this section is not aligned to left and doesn't have the same font size 

![figma form text](https://user-images.githubusercontent.com/59707859/229285255-2025bc0e-ca41-412f-b45f-bbe2c09ea2d0.PNG)
![student form text](https://user-images.githubusercontent.com/59707859/229285257-bde80c30-9340-49e1-993e-6afb29ea1463.PNG)

Reply:
Thank you for your review, all changes are done.